src/foam/u2/wizard/PrerequisiteWAO.js

```diff
@@ -83,18 +83,20 @@
 
       let clonedPrereqWizardletData;
 
-      if ( this.loadFromPropertyName  ){
-        if (  ! prereqWizardletData.hasOwnProperty(this.loadFromPropertyName) ){
+      if ( this.loadFromPath  ){
+        var loadedFromData = this.loadFromPath.f(prereqWizardletData);
+
+        if ( ! loadedFromData ){
           console.error(
-            `prerequisiteCapabilityId: ${this.prerequisiteCapabilityId}'s data does not have the property ${this.loadFromPropertyName}`
+            `prerequisiteCapabilityId: ${this.prerequisiteCapabilityId}'s data returns null for the path ${this.loadFromPath}`
```

Comment:
`this.loadFromPath` won't be a string anymore, so should probably be `this.loadFromPath.toSummary()`